docs/relier-communication-protocols/fx-webchannel.md

```diff
@@ -163,23 +166,45 @@ expected.
 ```
 
 * `customizeSync` - Should the user be shown the browser's `Customize Sync` dialog?
-* `declinedSyncEngines` - An array of Sync engines the user has deselected.  See [declinedSyncEngines](#declinedSyncEngines).
+* `declinedSyncEngines` - An array of Sync engines the user has deselected. See [declinedSyncEngines, offeredSyncEngines](#declinedSyncEngines-offeredSyncEngines).
 * `email` - User's email address.
 * `keyFetchToken` -
+* `offeredSyncEngines` - An array of Sync engines that were displayed to the user. See [declinedSyncEngines, offeredSyncEngines](#declinedSyncEngines-offeredSyncEngines).
 * `sessionToken` - The current session token that can be used to interact with FxA's auth server.
 * `uid` - The user identifier.
 * `unwrapBKey` - The key used to encrypt and decrypt the user's sync data.
 * `verified` - Is the user verified?
 * `verifiedCanLinkAccount` - Has the user verified they want to link accounts, if signing into a different account than previously signed in to?
 
-#### declinedSyncEngines
+#### capabilities
+An object containing browser supported capabilities. Only available with browser support.
 
-* bookmarks
-* history
-* passwords
-* tabs
-* desktop-addons
```

Comment:
The `desktop-` prefix was incorrectly added to the docs before. When logging against master and deselecting addons and preferences, this is what is sent in the WebChannel message:


<img width="334" alt="screen shot 2017-06-22 at 12 14 19" src="https://user-images.githubusercontent.com/848085/27431437-73ba3d10-5744-11e7-850d-8066a0666dc1.png">